mdstat.go

```diff
@@ -165,6 +178,13 @@ func parseMDStat(mdStatData []byte) ([]MDStat, error) {
 	return mdStats, nil
 }
 
+// check if a string's format is like the mdType
+// Rule 1: mdType should not be like (...)
+// Rule 2: mdType should not be like sda[0]
+func isRaidType(mdType string) bool {
```

Comment:
Ewww hah..
There must be a better way. Is there an explicit way to query the md type? E.g by reasing another procfs file?